front/src/components/UI/CompanyView/CompaniesSector.tsx

```diff
@@ -0,0 +1,24 @@
+import CompanyBlock from "./CompanyBlock";
+import { CompanyProps } from "./CompanyBlock";
+
+interface CompanySectorProps {
+  items: Array<CompanyProps>;
+}
+
+const Companies = (props: CompanySectorProps) => {
+  return (
+    <ul>
+      {props.items.map(
+        (company: { title: string; img_src: string; id: string }) => (
+          <CompanyBlock
```

Comment:
Tutaj jest bardzo istotny błąd o którym React informuje dopiero w konsoli (jak otworzysz to zobaczysz error typu: "Each child should have unique key").
Rozwiązanie jest proste: przekazać jako props parametr key={tutaj jakaś unikalna wartość dla każdego elementu z listy}

Często pytanie o to dlaczego klucze są potrzebne pojawia się na różnych rozmowach kwalifikacyjnych itd więc warto wiedzieć nie tylko jak naprawić ten błąd ale też dlaczego trzeba to zrobić i w jaki sposób to pomaga (w tym kursie od Maxa na Udemy jest to wyjaśnione)